dataactvalidator/filestreaming/csvS3Reader.py

```diff
@@ -24,18 +18,23 @@ def openFile(self,region,bucket,filename,csvSchema,bucketName,errorFilename):
         s3Bucket = s3connection.lookup(bucket)
         self.s3File = s3Bucket.lookup(filename)
         self.isLocal = False
-        if(self.s3File == None):
-            raise ValueError("".join(["Filename provided not found on S3: ",str(filename)]))
+        if not self.s3File:
+            raise ValueError("Filename {} not found on S3".format(filename))
 
-        super(CsvS3Reader,self).openFile(region,bucket,filename,csvSchema,bucketName,errorFilename)
+        super(CsvS3Reader, self).openFile(
+            region, bucket, filename, csvSchema, bucketName, errorFilename)
 
-    def downloadFile(self,region,bucket,filename,targetLocation):
+    def downloadFile(self, region, bucket, filename, targetLocation):
         """ After opening a file, download entire file to filename """
         s3connection = boto.s3.connect_to_region(region)
         s3Bucket = s3connection.lookup(bucket)
+        if not s3Bucket:
+            raise ValueError("Bucket {} not found in region {}".format(
+                bucket, region))
         self.s3File = s3Bucket.get_key(filename)
-        if(self.s3File == None):
-            raise ValueError("".join(["Filename provided not found on S3: ",str(filename)]))
+        if not self.s3File:
+            raise ValueError("Filename {} not found in bucket {}".format(
+                filename, bucket))
```

Comment:
This function is almost identical to `openFile`; would it make sense for one to call the other or to pull out the shared code into a `initializeS3File` method?